### PR TITLE
Enable mute requests from sip gateway.

### PR DESCRIPTION
**Pull request by theunafraid**

### CODE REVIEW

**Comment by damencho** [comment]
Jenkins, add to whitelist, please!

**Comment by damencho** [comment]
You want to put that behind flag, I suppose?

**Comment by theunafraid** [comment]
> You want to put that behind flag, I suppose?

Added flag for mute request.

File: src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java [hunk 1]
```diff
@@ -2294,11 +2294,9 @@ boolean handleMuteRequest(Jid fromJid,
             return false;
         }
 
-        // we do not want to remotely mute jigasi as currently
-        // there is no way for it to unmute
-        if (doMute && participant.isSipGateway())
+        if (doMute && (globalConfig.isSipMuteEnabled() == false))
```
Comment (damencho) [inline src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: Tests are failing. There is an error in the logic I think ... Isn't this the correct if statement:
```
if (doMute && participant.isSipGateway() && !globalConfig.isSipMuteEnabled())
```
?

Comment (damencho) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: You are not checking this whether it is jigasi and you always forbid remote muting, which tests caught.

Comment (theunafraid) [reply src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java]: Corrected condition for jigasi

**Pull request merged**